src/serac/physics/utilities/equation_solver.hpp

```diff
@@ -21,7 +21,7 @@
 #include "serac/infrastructure/input.hpp"
 #include "serac/physics/utilities/solver_config.hpp"
 
-namespace serac {
+namespace serac::mfem_ext {
```

Comment:
I'm not sure `EquationSolver` should be in this namespace. I can't imagine this getting upststreamed to MFEM as it is a convenient wrapper of MFEM stuff for us.

Reply:
My thinking was that because it extends an MFEM class it uses MFEM naming conventions and that we should keep it in the same namespace for consistency, i.e., MFEM naming **iff** in `mfem_ext`.

Reply:
It is a convenience wrapper around MFEM types that I doubt will get upstreamed, so I'm on the side of it shouldn't be in `mfem_ext`. However, I don't feel too strongly about this and will defer to the group consensus. Any thoughts @white238 @samuelpmishLLNL @jonwong12 ?